src/app/meme-dialog/meme-dialog.component.html

```diff
@@ -15,7 +13,25 @@
       <button mat-icon-button (click)="onClickDownVote()" [color]="myVote == -1 ? 'warn': 'basic'"><i
         class="material-icons">thumb_down</i></button>
     </div>
-
+    <mat-list>
+      <mat-divider></mat-divider>
+      <mat-list-item *ngFor="let comment of comments">
+        <div class="comment-line">
+          <div class="comment">
+            <div class="comment-info" matLine>{{ comment.User.username }} commented {{ comment.createdAt | timeAgo }}</div>
+            <div matLine>{{ comment.text }}</div>
+          </div>
+          <div class="delete-button" *ngIf="userService.user && comment.User.username===userService.user.username">
+            <button mat-icon-button (click)="deleteComment(comment.id)"><i class="material-icons">delete</i></button>
+          </div>
+        </div>
+        <mat-divider [inset]="true" ></mat-divider>
+      </mat-list-item>
+    </mat-list>
+    <form [formGroup]="form" (ngSubmit)="submitComment()" *ngIf="userService.isLoggedIn()">
```

Comment:
I think this should be above the comments, otherwise you have to scroll way down to the bottom.